src/modelmanager.cpp

```diff
@@ -541,10 +541,10 @@ bool ModelManager::configFileReloadNeeded() {
     struct stat statTime;
 
     stat(configFilename.c_str(), &statTime);
-    if (lastConfigChangeTime != statTime.st_ctime) {
-        return true;
+    if (configFilename == "" || (lastConfigChangeTime == statTime.st_ctime)) {
+        return false;
```

Comment:
`stat` probably shouldn't be even called when `configFilename == ""` & the same holds true for
`configFileReloadNeded` when there is no file to check.